scripts/thankyou.coffee

```diff
@@ -1,19 +0,0 @@
-# Description:
-#   Thank You
-#
-# Dependencies:
-#   None
-#
-# Configuration:
-#   None
-#
-# Commands:
-#
-# Author:
-#   Scumbag Joe
-
-module.exports = (robot) ->
-  robot.hear /\(thank you\)|(\bthank you\b)/i, (msg) ->
-    if msg.match[1] is 'thank you'
-      if Math.random() > .8
```

Comment:
This is where the probability comes in. Looks like this is doing it 20% of the time?